src/local-editor-service.ts

```diff
@@ -140,6 +139,24 @@ export class LocalEditorService extends EditorService {
     };
   }
 
+  _generateAutoCompletionForElement(e: Element): string {
+    let autocompletion = `<${e.tagName}`;
+    let tabindex = 1;
+    if (e.attributes.length) {
+      autocompletion += ` $${tabindex++}`;
```

Comment:
> This resolves to e.g. "$1" but I think the correct syntax that editors expect is "${1}".

In Atom `$1` and `${1}` are equivalent, but I will update this for consistency.

> What about this instead?

Yes agreed. After I wrote the insertionPoints logic I realized without is cleaner.

> This also highlights that this feature needs tests.

Correct. Just wanted to submit the PR since there was no corresponding issue discussion. E.g. to make sure we all agree on the function, before writing tests.